.github/workflows/ci.yml

```diff
@@ -57,3 +61,61 @@ jobs:
             make kind-tekton
         - name: Test
           run: make test-integration
+    e2e:
+        strategy:
+          matrix:
+            go-version: [1.15.x]
+            os: [ubuntu-latest]
+            kubernetes:
+              - v1.18.15
+              - v1.19.7
+              - v1.20.2
+        runs-on: ${{ matrix.os }}
+        steps:
+        - name: Install Go
+          uses: actions/setup-go@v2
+          with:
+            go-version: ${{ matrix.go-version }}
+        - name: Check out code
+          uses: actions/checkout@v2
+        - name: Install kubectl
+          uses: azure/setup-kubectl@v1
+          with:
+            version: ${{ matrix.kubernetes }}
+        - name: Create kind cluster
+          uses: helm/kind-action@v1.1.0
+          with:
+            version: v0.10.0
+            node_image: kindest/node:${{ matrix.kubernetes }}
+            cluster_name: kind
+            wait: 120s
+        - name: Verify kind cluster
+          run: |
+            echo "# Using KinD context..."
+            kubectl config use-context "kind-kind"
+            echo "# KinD nodes:"
+            kubectl get nodes
+        - name: Install Tekton
+          run: |
+            make kind-tekton
+            kubectl -n tekton-pipelines rollout status deployment tekton-pipelines-controller --timeout=1m
+            kubectl -n tekton-pipelines rollout status deployment tekton-pipelines-webhook --timeout=1m
+        - name: Install Registry
+          run: |
+            kubectl apply -f test/data/registry.yaml
+            kubectl -n registry rollout status deployment registry --timeout=1m
+        - name: Install ko
+          run: |
+            curl -L https://github.com/google/ko/releases/download/v0.8.0/ko_0.8.0_Linux_x86_64.tar.gz | tar xzf - ko
+            sudo mv ko /usr/local/bin
+        - name: Install Shipwright Build
+          run: |
+            make install-operator-kind
+            kubectl -n build-operator rollout status deployment build-operator --timeout=1m || true
+        - name: Test
+          run: TEST_E2E_OPERATOR=managed_outside TEST_NAMESPACE=build-operator TEST_IMAGE_REPO=registry.registry.svc.cluster.local:32222/shipwright-io/build-e2e make test-e2e
```

Comment:
does this means we should remove the https://github.com/shipwright-io/build/blob/master/hack/install-registry.sh ?

Reply:
Good question. In general, I prefer to run the registry in the cluster. But, in the testing.md doc when running the e2e tests locally, we still describe how to use the shell script. So, I would retain it for the time being. Or we update that section as well.

Reply:
sounds good